data/json/snippets.json

```diff
@@ -246,6 +246,166 @@
   },
   {
     "type": "snippet",
+    "category": "flier",
+    "text": [ "This is a flier for red ryder bbguns.  On it a child is pulling a shining red wagon with a cooked pheasant on it and a wooden rifle over one shoulder.  the child has a dog trailing beside him and a satisfied look on his face.  The caption read \"When you chose Red Ryder, you invest in the American Dream.  You invest in our Independence.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an old flier for a movie from the 30s.  A tan man with slick black hair and muscles bulging through his offwhite suit is clasping a woman to his hip with one hand, and the woman is wearing a black leather dress.  With her hips splayed, she is holding a pistol in one hand and starring into directly out.   The caption reads \"Witness the rebirth of New Noir with 'Jersey Shore Blues'.  Starring Jenifer Languiz as 'Snookie'!\"" ]
```

Comment:
"Staring into directly out" - it's the first time I see an expression like that. I don't think it's correct.